parkings/migrations/0031_default_domain.py

```diff
@@ -20,7 +20,9 @@ class Migration(migrations.Migration):
         migrations.AlterField(
             model_name='parking',
             name='domain',
-            field=models.ForeignKey(on_delete=django.db.models.deletion.PROTECT, related_name='parkings', to='parkings.EnforcementDomain'),
+            field=models.ForeignKey(on_delete=django.db.models.deletion.PROTECT, related_name='parkings',
+                to='parkings.EnforcementDomain', blank=True, null=True,
+            ),
```

Comment:
Is this AlterField still needed? What it changes (compared to previous version of the same field)?

Reply:
The previous version is here:

https://github.com/City-of-Helsinki/parkkihubi/blob/e19d6777a7c08edae4bb94107807a08a35c83bd0/parkings/migrations/0029_enforcement_domain.py#L75-L79

It seems that only difference is blank=True, but that can as well be left out. So, this whole AlterField command can be removed from this migration.